src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py

```diff
@@ -78,10 +78,19 @@ def compute_clip_text_embedding(self, text: str, negative_text: str = "") -> Ten
         """
         conditional_embedding = self.clip_text_encoder(text)
         if text == negative_text:
-            return torch.cat(tensors=(conditional_embedding, conditional_embedding), dim=0)
+            negative_embedding = conditional_embedding
+        else:
+            if isinstance(text, list) and isinstance(negative_text, list):
+                assert len(text) == len(
+                    negative_text
+                ), "The length of the text list and negative_text should be the same"
+
+            if isinstance(negative_text, str) and isinstance(text, list):
+                negative_text = [negative_text] * len(text)
+
+            negative_embedding = self.clip_text_encoder(negative_text)
 
-        negative_embedding = self.clip_text_encoder(negative_text or "")
-        return torch.cat(tensors=(negative_embedding, conditional_embedding), dim=0)
+        return cat((negative_embedding, conditional_embedding))
```

Comment:
Let's make it more compact:
```py
        text = [text] if isinstance(text, str) else text
        negative_text = [negative_text] if isinstance(negative_text, str) else negative_text
        assert len(text) == len(negative_text), "The length of the text list and negative_text should be the same"
        conditional_embedding = self.clip_text_encoder(text)
        negative_embedding = self.clip_text_encoder(negative_text) if negative_text else conditional_embedding
        return cat((negative_embedding, conditional_embedding))
 ```
(or smthg like that)

Reply:
done